packages/block-library/src/button/edit.native.js

```diff
@@ -352,7 +395,13 @@ class ButtonEdit extends Component {
 								styles.outline,
 								{
 									borderRadius: outlineBorderRadius,
-									borderWidth: styles.button.borderWidth,
+									// Border Width has to be spread since width is thicker
+									// on Android when border radius is decreased to 0
+									borderLeftWidth: styles.button.borderWidth,
+									borderRightWidth: styles.button.borderWidth,
+									borderTopWidth: styles.button.borderWidth,
+									borderBottomWidth:
```

Comment:
How about move border style for Android to class in `editor.native.scss` e.g. .`buttonBorderAndroid` ?

then:
**`edit.native.js`**
```jsx
style={ [
   styles.outline,
   styles.buttonBorderAndroid,
   {
      borderRadius: outlineBorderRadius,
      borderColor: backgroundColor,
   },
] }
```

**`edit.native.scss`**
```css
.buttonBorderAndroid {
	border-top-width: $border-width;
	border-right-width: $border-width;
	border-bottom-width: $border-width;
	border-left-width: $border-width;
}
```